libraries/core_libs/consensus/include/pbft/pbft_manager.hpp

```diff
@@ -149,8 +156,9 @@ class PbftManager : public std::enable_shared_from_this<PbftManager> {
   std::shared_ptr<PbftBlock> getUnfinalizedBlock_(blk_hash_t const &block_hash);
 
   std::atomic<bool> stopped_ = true;
-  // Using to check if PBFT block has been proposed already in one period
-  std::pair<blk_hash_t, bool> proposed_block_hash_ = std::make_pair(NULL_BLOCK_HASH, false);
+
+  // Only be able to propose one PBFT block for each period
```

Comment:
weird wording.... -> Ensures that only one PBFT block per period can be proposed ?

Reply:
Fixed, thanks